src/components/common/DropDownMenu/dropDownMenu.tsx

```diff
@@ -0,0 +1,125 @@
+import { font } from "@/styles/font";
+import { useState } from "react";
+import { color } from "@/styles/color";
+import styled, { css } from "styled-components";
+import ArrowDropDownMenu from "../Icon/ArrowDropDownMenu";
+
+interface DropdownItemType {
+  dropdownItemText: string;
+}
+
+interface PropsType {
+  desc: string;
+  dropdownMenuDatas: DropdownItemType[];
+  width: string;
+}
+
+const DropDownMenu = ({
+  desc,
+  dropdownMenuDatas,
+  width = "320px",
+}: PropsType) => {
+  const [isOpenDropdownMenu, setIsOpenDropdownMenu] = useState(false);
+  const [dropdownMenuText, setDropdownMenuText] =
+    useState("옵션을 선택해 주세요");
+
+  const clickedMenu = (item: DropdownItemType) => {
+    setDropdownMenuText(item.dropdownItemText);
+    setIsOpenDropdownMenu(false);
+  };
+
+  const clickedToggle = () => {
+    setIsOpenDropdownMenu((prev) => !prev);
+  };
+  return (
+    <StryledDropdownMenu style={{ width }}>
+      <Desc>{desc}</Desc>
+      <DropdownMenuBox onClick={clickedToggle} isOpen={isOpenDropdownMenu}>
+        <DropdownMenuText>{dropdownMenuText}</DropdownMenuText>
+        <ArrowDropDownMenu />
+      </DropdownMenuBox>
+      <DropdownMenuListBox isOpen={isOpenDropdownMenu}>
+        <DropdownMenuList>
+          {dropdownMenuDatas?.map((item, index) => (
+            <DropdownMenuItem
+              key={`dropdown ${index}`}
+              onClick={() => clickedMenu(item)}
+            >
+              {item.dropdownItemText}
+            </DropdownMenuItem>
+          ))}
+        </DropdownMenuList>
+      </DropdownMenuListBox>
+    </StryledDropdownMenu>
+  );
+};
+
+export default DropDownMenu;
+
+const StryledDropdownMenu = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 320px;
+  gap: 8px;
+`;
+
+const Desc = styled.p`
+  ${font.context}
+`;
```

Comment:
```suggestion
const Desc = styled.p`
  ${font.context}
  color: ${color.gray700};
  padding-bottom: 8px;
`;
```

색깔이랑 패딩이 안들어간것같습니다 확인 부탁드립니다 ~